.eslintrc.json

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
I would have kept the configuration as YAML, personally.  (I don't see any commit message explaining why it was changed to JSON, if there's a technical reason.)

Reply:
It was a bit hard for me to understand YAML hence switched to JSON then. Reverted back with the updates now :)